hack/make/test-buildkit

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -e -o pipefail
+
+source hack/make/.integration-test-helpers
+
+if [ ! -z "${TEST_SKIP_BUILDKIT}" ]; then
+	echo buildkit tests skipped according to TEST_SKIP_BUILDKIT
+	exit 0
+fi
+
+(
+	env
+	bundle .integration-daemon-start
+
+	export BIN_PATH="${PWD}/bundles/dynbinary-daemon"
+	export PATH="${BIN_PATH}:${PWD}/bundles/binary-daemon:$PATH"
+
+	# Download buildkit
+	# TODO: change this to moby version of the project after
+	# https://github.com/moby/buildkit/pull/1164 is merged.
+	echo "curl -fsSL -o ${PWD}/buildkit.tar.gz https://github.com/SamWhited/buildkit/archive/${BUILDKIT_COMMIT}.tar.gz"
+	curl -fsSL -o ${PWD}/buildkit.tar.gz https://github.com/SamWhited/buildkit/archive/${BUILDKIT_COMMIT}.tar.gz
+	tar -xzf ${PWD}/buildkit.tar.gz
+	echo "GOPATH: $GOPATH"
+	mv buildkit-${BUILDKIT_COMMIT}/ $GOPATH/buildkit/
+	cd $GOPATH/buildkit/
+
+	function cleanup {
+		bundle .integration-daemon-stop
+		cleanup_test_suite_binaries
+		error_on_leaked_containerd_shims
+	}
+	# Always run cleanup, even if the command fails.
+	trap cleanup EXIT
+	
+	export PREFER_LEGACY=1
```

Comment:
(just wondering) do these have to be exported, or would just putting these before `./hack test` do the trick?

Reply:
They'll work before `./hack test`, I think I just moved them so the line wasn't as long, but it doesn't really matter and I can use line continuation for that. Will move them back: tightly limiting the scope is more important than minor style tweaks.

Reply:
Actually; given that it runs in a sub shell, I don't think scope is an issue (but won't hurt either) 🤷‍♂